### PR TITLE
Form implementation

### PR DESCRIPTION
**Pull request by anton25360**

### CODE REVIEW

**Comment by anton25360** [comment]
Only form has been added, no data manipulation yet.

File: index.php [hunk 1]
```diff
@@ -25,15 +27,28 @@
     echo '<p> Event: ' . $event . ', '. $year . '. You came ' . $place . $message . '</p>';
 }
 
+echo addEvent($_POST['event'],$_POST['year'],$_POST['place']);
```
Comment (georginaw) [inline index.php]: Where does $_POST data come from if on same page as display (when user first lands on index.php the fields will be empty?)

File: index.php [hunk 2]
```diff
@@ -25,15 +27,28 @@
...
     <link href="normalize.css" rel="stylesheet" type="text/css"> <!-- normalize file -->
     <link href="style.css" rel="stylesheet" type="text/css"> <!-- css file -->
     <link href="assets/webfonts/stylesheet.css" rel="stylesheet" type="text/css"> <!-- font file -->
-
 </head>
 
 <body>
+
+<br>
+<div class="wrapper">
```
Comment (georginaw) [inline index.php]: Do you need a div to put the form in?

Comment (anton25360) [reply index.php]: nope- removed

**Pull request merged**